flink/README.md

```diff
@@ -18,7 +18,7 @@ No additional installation is needed on your server.
 
 1. Configure the [Datadog HTTP Reporter][4] in Flink.
 
-     Copy `<FLINK_HOME>/opt/flink-metrics-datadog-1.9.2.jar` into your `<FLINK_HOME>/lib` folder. In your `<FLINK_HOME>/conf/flink-conf.yaml`, add these lines, replacing `<DATADOG_API_KEY>` with your Datadog [API key][10]:
+     Copy `<FLINK_HOME>/opt/flink-metrics-datadog-<VERSION>.jar` into your `<FLINK_HOME>/lib` folder. In your `<FLINK_HOME>/conf/flink-conf.yaml`, add these lines, replacing `<DATADOG_API_KEY>` with your Datadog [API key][10]:
```

Comment:
Is this the version for the datadog reporter? Can we specify that, it might be unclear if it's the version for flink or datadog.

Reply:
```suggestion
     Copy `<FLINK_HOME>/opt/flink-metrics-datadog-<DATADOG_REPORTER_VERSION>.jar` into your `<FLINK_HOME>/lib` folder. In your `<FLINK_HOME>/conf/flink-conf.yaml`, add these lines, replacing `<DATADOG_API_KEY>` with your Datadog [API key][10]:
```
or something along those lines

Reply:
Yes, it is the reporter version!